web/index.html

```diff
@@ -382,13 +383,13 @@
       </header>
       <div id="layer-modal-main">
         <div id="layer-categories">
-          
+
         </div>
         <div id="selected-category">
-          
+
```

Comment:
I'd leave out these spaces if there isn't anything inside the divs `<div></div>`